testsuite/tests/typing-jkind-bounds/composite.ml

```diff
@@ -520,13 +570,7 @@ Error: This value is "contended" but expected to be "uncontended".
 type 'a t : immutable_data = Flat | Nested of 'a t t
 (* CR layouts v2.8: This should work once we get proper subsumption. *)
 [%%expect {|
-Line 1, characters 0-52:
-1 | type 'a t : immutable_data = Flat | Nested of 'a t t
-    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-Error: The kind of type "t" is value
-         because it's a boxed variant type.
-       But the kind of type "t" must be a subkind of immutable_data
-         because of the annotation on the declaration of the type t.
+type 'a t = Flat | Nested of 'a t t
```

Comment:
Is this in-progress work? This test seems to be failing now (on my branch too) in favor of the old output